app/help/en/faq.md

```diff
@@ -64,182 +1109,261 @@ https://www.linkedin.com/in/nedscott
 Daniel Larimer, CTO of Steemit, @dan @dantheman
 https://www.linkedin.com/in/daniel-larimer-0a367089
 
-<a href="/faq.html#Table_of_Contents">Table of Contents</a>
-<a class="anchor" name="What_is_the_difference_between_Steem_and_Steemit"></a>
-## What is the difference between Steem and Steemit?
+<a href="/faq.html#Table_of_Contents_Steemit__Inc">&#11025;</a>
+<a class="anchor" name="How_does_Steemit__Inc__earn_money"></a>
+## How does Steemit, Inc. earn money?
 
-Steem is the name of both the database and digital points, which are a form of cryptocurrency that Steemit.com plugs into. 
+Over time, Steemit will allow advertisers and bloggers to promote content by buying and burning SBD/STEEM. Steemit can benefit from sales of Steem to advertisers in the cryptocurrency markets or offer advanced services to these advertisers and bloggers.
```

Comment:
we do not have any plans at this time to allow advertising on the site, and may never do so.  please remove the term "advertisers" from the FAQ.